html-css/lesson1.md

```diff
@@ -44,15 +46,18 @@ We'll cover the following semantic tags:
   - ``<article>``
   - ``<aside role="complementary">``
 
-> **Exercise (pair programming):**  Work in pairs to determine where to place these new Tags and Attributes in the index.html file. Who benefits when we write "semantic" HTML?
+> **Exercise (pair programming):**  Switch between the person who is driving and the one who is helping out. The person who is now driving will `fork` the repository and `clone` it to their computer. Add the gray section with the text 'Bikes for Refugees' and the two buttons. Update the website to use semantic tags.
```

Comment:
The gray section is already there, so maybe we can just keep the "semantic" part?